jyulb/internal/common/tests/test_proxy_lattice.py

```diff
@@ -0,0 +1,133 @@
+"""Testing module for jyu-lb proxy-lattice data class."""
+import unittest
+import numpy as np
+import numpy.testing as np_test
+from simphony.core.cuba import CUBA
+from simphony.cuds.lattice import LatticeNode
+from simphony.cuds.abstractlattice import ABCLattice
+from simphony.core.data_container import DataContainer
+from jyulb.internal.common.proxy_lattice import ProxyLattice
+from jyulb.internal.common.domain import PyLattice, PyGeometry
+from jyulb.internal.common.domain import PyIsothermalData
+
+
+class ProxyLatticeTestCase(unittest.TestCase):
```

Comment:
I tried this, but there are some complications related to **Issue 80** in simphony-common (applies not only to the size, origin, and base_vect but also to the index arguments). Namely, the ABCCheckLattice executes tests with 2D lattices (e.g. *test_get_coordinate_square*) where 2D parameters and indices are utilized.

It appears cumbersome to accommodate these tests with ProxyLattice (which currently assumes 3D cubic lattice) ...

About the issue 80: should we also enforce 3D indices for *get_node* etc.?